<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewUI</class>
 <widget class="QDialog" name="ViewUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>151</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rodytiButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Rodyti</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="iseitiButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>71</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>IÅ¡eiti</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
